lib/core/search-text/search-text-input.component.ts

```diff
@@ -133,11 +149,13 @@ export class SearchTextInputComponent implements OnInit, OnDestroy {
                     this.subscriptAnimationState = this.toggleAnimation();
                     if (this.subscriptAnimationState.value === 'inactive') {
                         this.searchTerm = '';
+                        this.searchChange.emit('');
```

Comment:
not sure about that... we already have the reset... probably u should do something with that event instead of relying on the searchChange